apis/swagger.yml

```diff
@@ -351,6 +351,30 @@ paths:
           schema:
            $ref: "#/definitions/Error"
       tags: ["Container"]
+  /containers/{id}/pause:
+    post:
+      summary: "Pause a container"
+      operationId: "ContainerPause"
+      parameters:
+        - name : "id"
+          in : "path"
+          required: true
+          description: "ID or name of the container"
+          type: "string"
+      responses:
+        204:
```

Comment:
in code above, you add code like this: `resp.WriteHeader(http.StatusOK)`

But there in swagger.yml, you defined 204 as a successful response. Maybe we need to be consistent.